tests/test_search.py

```diff
@@ -149,3 +46,35 @@ def test_search_with_multiple_lines(self, mozwebqa):
         cs_super.click_search()
 
         Assert.true(cs_super.are_search_results_found)
+
+
+class TestSearchForSpecificResults:
+
+    @pytest.mark.nondestructive
+    def test_that_search_for_valid_signature(self, mozwebqa):
```

Comment:
This method name is unclear. Do you mean simply `test_search_for_valid_signature` or should it be `test_that_search_for_valid_signature`[something here]. Currently the `that` in the name doesn't really make sense.